### PR TITLE
capstone project

### PR DESCRIPTION
**Pull request by Said-laasri**
in this PR I added all files of the project the main page HTML and about page HTML and all the styling in the CSS file .
in the javascript file, I added 2 js feature the popup menu and javascript HTML injection for the section of speakers.
all using the requested font family and color in the guideline design.
and to station all the sections in HTML I used flexbox and grid CSS
and using icons from awesome icon and Flaticone and images from Feekpik "website"

this website is an invitation to the professionals to our services such as import/export and transit
*****************************************************************************************************************************************
This is a small video presenting the Project ====> LINK:https://www.loom.com/share/6ccceca989034e51a84f1c6f59cbde7c

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            <div class="tags"><a href="#">Join</a></div>
+            <div class="tags"><a href="#partners">Sponsor</a></div>
+            <div class="tags"><a href="#post-card">News</a></div>
+            <div class="tags"><a id="efx" href="about.html">About</a></div>
+        </div>
+    </nav>
+    <section class="intro">
+        <article id="Welcome-about" class="Welcome">
+            <h3 id="about-h3">"Hello! Sharing World"</h3>
+            <h1 id="about-h1">LOGISTICS INT DAY IN NEW SUIT! 2022</h1>
```
Comment (leonardodiasb) [inline about.html]: - [x] `Hx` headers should follow a hierarchy and `h3` should not come before the `h1` tag. Kindly use appropriate tags to write a text before the `h1` tag.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        <div class="menu">
+            <div class="closer"><i class="fa-solid fa-xmark"></i></div>
+            <div class="tags"><a id="edx" href="index.html">Home</a></div>
+            <div class="tags"><a href="#Program-cards">Program</a></div>
+            <div class="tags"><a href="#">Join</a></div>
+            <div class="tags"><a href="#partners">Sponsor</a></div>
+            <div class="tags"><a href="#post-card">News</a></div>
+            <div class="tags"><a id="efx" href="about.html">About</a></div>
+        </div>
+    </nav>
```
Comment (leonardodiasb) [inline about.html]: - [x] The top dark bar is part of the navbar. Kindly fix that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        </li>
+
+        <li>
+            <img src="images/logistic.png" alt="processing">
+            <h6>Sea-Trans</h6>
+            <p>As a first step of exporting product packaging is most important as it asures the security of product</p>
+        </li>
+
+        <li>
+            <img src="images/distribution.png" alt="processing">
```
Comment (leonardodiasb) [inline index.html]: - [x] `alt` attributes for images should have a clear description of the image for better understanding. Kindly use more descriptive words.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,730 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap');
```
Comment (leonardodiasb) [inline style.css]: - [x] You should stick with the fonts provided and used by the Template: Cocogoose, Lato

File: style.css [hunk 5]
```diff
@@ -0,0 +1,730 @@
...
+
+.minimenu {
+  display: flex;
+  background: linear-gradient(0deg, hsla(240, 15%, 45%, 1) 1%, hsla(240, 24%, 42%, 1) 27%, hsla(240, 29%, 41%, 1) 50%, hsla(240, 53%, 35%, 1) 74%, hsla(204, 77%, 76%, 1) 99%, hsla(240, 4%, 57%, 1) 100%);
+  width: 100%;
+  height: 100rem;
+  position: relative;
+  flex-direction: column;
+  margin-top: -100px;
+}
```
Comment (leonardodiasb) [inline style.css]: - [x] The mobile menu should follow the design of the page and use similar font families and colors: 
- Font families: Cocogoose, Lato.
- Color swatch: #272A31, #EC5242, #D3D3D3

File: about.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+            <hr class="line">
+            <p>The log of CC Global Summit 2022 was decided throught the logo compitition from 8.June to 7.July.</p>
+            <img src="images/logo44.png" alt="logo-website">
+        </article>
+        <div class="past-summits">
+        <article>
+            <h4>See the past CC Global Summits</h4>
+            <hr class="line">   
+            <p>Take a look at the last CC Global Summits wich took place in Buenos Aires and Warsaw</p>
+        </article>
```
Comment (leonardodiasb) [inline about.html]: - [x] The about page is divided into 3 different sections. Kindly change the tag names to keep the correct HTML structure.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Said-laasri , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README is descriptive ✔️

## Aspects to improve ♻️

- [x] Your PR description is not clear and professional. Please describe the changes and features you added.
- [x] You did not use the proper font families and colors from the Template for your whole project. Kindly change the colors to match the basic color swatch and use the original font family.
  - Basic color swatch: #272A31, #EC5242, #D3D3D3
  - Fonts used originally: Cocogoose, Lato
- [x] The video should contain a piece of code you are proud of.
  => I suggest you submit another video that fulfills all of the video presentation requirements.
- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Said-laasri** [comment]
thanks for the colours and fonts i didn't even notice that thanks for appointing that

File: about.html [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+      </article>
+    </section>
+    <section>
+      <article class="subject">
+        <h2>Creative Commons Global Summit 2022 Logo</h2>
+        <hr class="line">
+        <p>The log of CC Global Summit 2022 was decided throught the logo compitition from 8.June to 7.July.</p>
+        <img src="images/logo44.png" alt="logo-website">
+      </article>
+    </section>
```
Comment (danmainah) [inline about.html]: - [x] @Said-laasri  great work implementing the changes, Kindy re-configure this code, it supposes to belong to one section check the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)  the sections are only three I can see you have several sections.

- [x] [optional] Kindly check the spelling of your pull request title, in the description, you could add what the project is about currently it's more of your code and less of what it achieved.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Said-laasri , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 7 above)
Comment (Said-laasri) [reply about.html]: @danmainah   Thank you for reviewing my project, I will make the necessary corrections as suggested

File: about.html [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+        <div>
+          <a href="#">My Page</a>
+        </div>
+        <div>
+          <a href="#">Logout</a>
+        </div>
+      </div>
+      <div class="seconde-bar">
+        <a class="Logo" href="index.html">
+          <img src="images/logo44.png" alt="logo-website">
```
Comment (prolajumokeoni) [inline about.html:41]: ### OPTIONAL 

- [x] Kindly reduce logo size to look similar to the Behance design 👍🏿 

Behance Design          |  Your design    
:-------------------------:|:-------------------------:
Screenshot_265|  Screenshot_266

**Review by prolajumokeoni — APPROVED** [review]
Hi @Said-laasri, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀 
- Good design 💯 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

all-done-gif


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 8 above)
Comment (Said-laasri) [reply about.html:41]: @prolajumokeoni  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: about.html (hunk 1 above)
Comment (Said-laasri) [reply about.html]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: about.html (hunk 2 above)
Comment (Said-laasri) [reply about.html]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: index.html (hunk 3 above)
Comment (Said-laasri) [reply index.html]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: style.css (hunk 4 above)
Comment (Said-laasri) [reply style.css]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: style.css (hunk 5 above)
Comment (Said-laasri) [reply style.css]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

File: about.html (hunk 6 above)
Comment (Said-laasri) [reply about.html]: @leonardodiasb  Thank you for reviewing my project, I will make the necessary corrections as suggested

**Comment by Said-laasri** [comment]
@prolajumokeoni  Thank you for reviewing my project.

**Pull request merged**